dashboard/app/views/levels/_list.html.haml

```diff
@@ -87,7 +87,7 @@
 
         -# Level name column, linked to view action if permitted
         %td
-          - if can? :view, level
+          - if can? :show, level
```

Comment:
:man_facepalming: Why are these different?

Reply:
I don't think `:view` means anything. there is `:read`, perhaps?